src/en/users-models.md

```diff
@@ -0,0 +1,78 @@
+Title: Juju users and models
+TODO: Stuff on user-added models (ssh key and credentials)
+
+
+# Users and models
+
+This section is about understanding models with multiple users.
+
+
+## Adding models
+
+When an administrator adds a model, by default, the cloud credentials used
+throughout the model will be those that the admin used to create the controller
+and the SSH keys copied across the model will be those of the admin model. The
```

Comment:
they will be when i add stuff about user-added models. i'd like to keep this in here for a while